taxi/forms.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from taxi.models import Driver, Car
+from django.contrib.auth import get_user_model
+
+
+class DriverCreationForm(UserCreationForm):
+  class Meta(UserCreationForm.Meta):
+    model = Driver
+    fields = UserCreationForm.Meta.fields + (
+      "first_name",
+      "last_name",
+      "license_number", 
+    )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+  class Meta:
+      model = Driver
+      fields = ("license_number",)
```

Comment:
The `DriverLicenseUpdateForm` should validate the `license_number` to ensure it consists of 8 characters, with the first 3 being uppercase letters and the last 5 being digits. Consider using the `validate_license_number` function from `validators.py` to enforce this validation.